mock-builder/tests/pallet.rs

```diff
@@ -197,6 +198,26 @@ mod test {
 		});
 	}
 
+	#[test]
+	fn basic_several_times() {
+		System::externalities().execute_with(|| {
+			let handler1 = MockTest::mock_qux(|p1| &p1 == "hello");
```

Comment:
Would it make more sense to have something like `MockTest::mock_qux(|p1| &p1 == "hello").times(x)`, and when the mock is dropped to confirm that `qux` was called x times with `"hello"`? 

Similar to https://github.com/asomers/mockall?tab=readme-ov-file#usage

Reply:
Interesting!! Much more funcional, I like it!

Let me play with it. Not fully sure if I can implement this because the mock is not dropped until the whole thread is dropped.